src/corebluetooth.cs

```diff
@@ -43,6 +45,59 @@ interface CBCentralInitOptions {
 	interface CBManager {
 		[Export ("state", ArgumentSemantic.Assign)]
 		CBManagerState State { get; }
+
+		[iOS (13,0), TV (13,0), Watch (6,0), Mac (10, 15)]
+		[Export ("authorization", ArgumentSemantic.Assign)]
+		CBManagerAuthorization Authorization { get; }
+	}
+
+	[iOS (13,0), TV (13,0), Watch (6,0)]
+	[StrongDictionary ("CBConnectionEventMatchingOptionsKeys")]
+	interface CBConnectionEventMatchingOptions {
+		NSUuid [] PeripheralUuids { get; set; }
+		CBUUID [] ServiceUuids { get; set;}
+	}
+
+	[iOS (13,0), TV (13,0), Watch (6,0)]
+	[Static]
+	[Internal]
+	interface CBConnectionEventMatchingOptionsKeys {
+		[Field ("CBConnectionEventMatchingOptionPeripheralUUIDs")]
+		NSString PeripheralUuidsKey { get; }
+		[Field ("CBConnectionEventMatchingOptionServiceUUIDs")]
+		NSString ServiceUuidsKey { get; }
+	}
+
+	[StrongDictionary ("CBConnectPeripheralOptionsKeys")]
+	interface CBConnectPeripheralOptions {
+		[Mac (10,13)] [iOS (6,0)]
+		bool NotifyOnConnection { get; set; }
+		bool NotifyOnDisconnection { get; set; }
```

Comment:
that one has no availability ? looks weird (add comment if that's the case)
if possible move some (all) availability attributes on the type itself

Reply:
That is the only one that had no availability attributes from the old constant used: https://github.com/xamarin/xamarin-macios/blob/master/src/corebluetooth.cs#L105 

The other attributes I used from the already present (set to obsolete) constants. So I don't think I can move [Mac (10,13)] [iOS (6,0)] to the top of the class :/